packages/contract-types/dist/contract-types/src/index.d.ts

```diff
@@ -0,0 +1 @@
+export * as Contracts from "./generated";
```

Comment:
@dcposch do you know if there's a way to not commit these dist files but build them for other package includes? do we just need to add a publish pipeline to npm or something for that?

I think some yarn/npm workspace magic should also be able to do this automagically, but I can't find on googling easily